### DISCUSSION TITLE
Enhancing face:background_job Performance through GPU Parallelization

### DISCUSSION THREAD

**Discussion by stanke-sk** [matiasdelellis/facerecognition#727] [Ideas]
Hello Everyone,

First off, I want to express my gratitude to everyone who has contributed to this project. I have a question regarding the possibility of parallelizing the face:background_job on a GPU. I've noticed that my GPU utilization is quite low,nearly idling, and it's only using about 1344MB of memory with the analysis_image_area --value 6291456 setting as the highest allowed by the model 4. I'm looking to enhance the image processing speed further. I have the capability to employ even additional GPU cards if necessary. Is there a way to achieve this, or is it something that could be considered for future development? I'm eager to maximize GPU or GPUs' capabilities. Thank you!

**Comment by matiasdelellis** [comment]
There is a lot of progress in parallelism, but regarding the use of GPU you must compile dlib and pdlib yourself to take advantage of it.

https://github.com/matiasdelellis/facerecognition/commit/ecd603a9ae76c36e2408d403ae39aecb473d524e
